src/creature.cpp

```diff
@@ -614,15 +614,15 @@ void Creature::onCreatureMove(Creature* creature, const Tile* newTile, const Pos
 	}
 }
 
-std::vector<Creature*> Creature::getKillers()
+CreatureVector Creature::getKillers()
 {
-	std::vector<Creature*> killers;
+	CreatureVector killers;
 	const int64_t timeNow = OTSYS_TIME();
 	const uint32_t inFightTicks = g_config.getNumber(ConfigManager::PZ_LOCKED);
 	for (const auto& it : damageMap) {
 		Creature* attacker = g_game.getCreatureByID(it.first);
 		if (attacker && attacker != this && timeNow - it.second.ticks <= inFightTicks) {
-			killers.push_back(attacker);
+			killers.push_back(attacker); 
```

Comment:
Mistake?

Reply:
I think it is already solved in [1a46099](https://github.com/otland/forgottenserver/pull/3241/commits/1a4609923e6f3fc91fa042e4a4f0e78e207571a2)